Description: Spawn tracker-needle instead of tracker-search-tool
Author: Petr Gajdůšek <gajdusek.petr@centrum.cz>
Forwarded: https://launchpad.net/kupfer/+bug/1082467

Index: kupfer-0+v208/kupfer/plugin/tracker1.py
===================================================================
--- kupfer-0+v208.orig/kupfer/plugin/tracker1.py	2012-06-01 19:51:14.000000000 +0200
+++ kupfer-0+v208/kupfer/plugin/tracker1.py	2013-12-19 21:05:43.663853407 +0100
@@ -49,7 +49,8 @@
 		Action.__init__(self, _("Search in Tracker"))
 
 	def activate(self, leaf):
-		utils.spawn_async(["tracker-search-tool", leaf.object])
+		search_tool = utils.lookup_exec_path("tracker-needle") or "tracker-search-tool"
+		utils.spawn_async([search_tool, leaf.object])
 	def get_description(self):
 		return _("Open Tracker Search Tool and search for this term")
 	def get_icon_name(self):
